torax/tests/persistent_cache_test.py

```diff
@@ -109,59 +110,25 @@ def get_simulation_time(text):
     # the first simulation step.
     t0 = get_simulation_time(out0)
     t1 = get_simulation_time(out1)
-
-    # Number of seconds we expect the cache to save on the second run
-    # This rule is likely to need adjusting to support more machine types
-    # and more CI environments, or possibly may need adjusting for
-    # flakiness (in initial testing of this rule it passed 100 / 100 runs)
-    # so be suspicious if it becomes highly flaky without a good reason.
+    # on CI we require a non‑trivial speedup; locally compilation
+    # overhead may be too small to outperform the simulation time.
```

Comment:
We haven't observed this test being flaky on local development so far and do still expect some sort of speedup when using the persistent cache. It could be that the threshold needs adjusting (or perhaps made into a relative speedup threshold) but this would be a useful test to keep running. What are the timings you are seeing for the first and second simulation?